src/config/initializers/apn.ts

```diff
@@ -0,0 +1,146 @@
+import * as http2 from 'http2';
+import jwt, { JwtHeader } from 'jsonwebtoken';
+import Logger from '../../lib/logger';
+
+export default class Apn {
+  private static instance: Apn;
+  private client: http2.ClientHttp2Session;
+  private readonly maxReconnectAttempts = 3;
+  private reconnectAttempts = 0;
+  private reconnectDelay = 1000;
+  private readonly bundleId = process.env.APN_BUNDLE_ID;
+
+  private readonly apnUrl =
+    process.env.NODE_ENV === 'production' ? 'https://api.push.apple.com' : 'https://api.sandbox.push.apple.com';
```

Comment:
This should be an environment variable, not a ternary with hardcoded URLs, especially if we want to open the option to modify this values remotely without re-deployments